PHP/PHP.sublime-completions

```diff
@@ -1596,12 +1596,12 @@
 		},
 		{
 			"trigger": "calcul_hmac()",
-			"contents": "calcul_hmac(${1:clent}, ${2:siretcode}, ${3:price}, ${4:reference}, ${5:validity}, ${6:taxation}, ${7:devise}, ${8:language})",
+			"contents": "calcul_hmac(${1:client}, ${2:siretcode}, ${3:price}, ${4:reference}, ${5:validity}, ${6:taxation}, ${7:devise}, ${8:language})",
```

Comment:
This function appears to be long dead. It's intentionally undocumented. I'm offering this change because it seems like it's likely more useful spelling to an English reader.

For the curious, the source for this appears to be:
https://pecl.php.net/package/spplus/1.0

Which also has:

> /* {{{ proto string signerurlpaiement(string clent , string data) Getting a url for the paiement redirection. Note that data is: $data = "$codesiret$reference$langue$devise$montant$taxe$validite"; */

The code appears to be written in some poor mix of French and English given [`paiement`](
https://www.larousse.fr/dictionnaires/francais/paiement/57252)